### ISSUE TITLE
page scrolling not totally fixed in 1.5.1

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JustLoy77/ebookdroid#294]
```
What steps will reproduce the problem?
1.Open ebookdroid
2.Open one of the books on shelf (pdf of djvu)
3. Try to scroll up or down

What is the expected output?
Touching screen only once to scroll up or down.

What do you see instead?
I have to touch the screen twice to scroll up or down. This interrupts reading. 
After page starts scrolling, it's possible to continue scrolling normally, with 
just one touch. But if I stop scrolling and wait a few seconds, the page 
"locks" and I have to touch the screen twice to scroll again. This did not 
happen in previous versions.


What version of the product are you using?
1.5.1

On what operating system?
Android 4.1.1

What device are you using?
Motorola Xoom 1


Please provide any additional information below.
I reported a bug with scrolling on version 1.5.0 previously. Then I installed a 
developer version 1.5.1 that appeared to solve the issue. However, an automatic 
download to "official" version 1.5.1 seemed to bring part of the problem back! 
On previous versions of ebookdroid I never had this problem. I am a heavy user 
of ebookdroid and having to touch the screen twice to scroll is really 
annoying. I don't know if "locking" the page after a few seconds is an 
intentional feature of version 1.5.1. If it is, I think a new "settings" option 
to turn this feature off could be made available.

```

Original issue reported on code.google.com by `bruno.ma...@gmail.com` on 27 Aug 2012 at 2:56

**Comment by GoogleCodeExporter** [comment]
```
http://stackoverflow.com/questions/8236790/in-android-4-0-navigation-bar-hijack-
first-touch-event
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 2 Sep 2012 at 12:54
- Changed state: **Postponed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `Alexander.V.Kasatkin@gmail.com` on 2 Sep 2012 at 12:55
- Added labels: **fullscreen**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in trunk. Try r1588:
http://code.google.com/p/ebookdroid/downloads/detail?name=ebookdroid-1.5.2-dev-r
1588.apk
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 7 Sep 2012 at 1:15
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Version 1.5.6 on Galaxy Note II has this problem.

When "Full screen - hide status bar" is ON and you have not tapped or swiped 
for approx 30 seconds, the first tap or swipe is lost.
ie:
1. do nothing for 30 seconds
2. swipe to change page - nothing happens
3. swipe again immediately - page changes
4. swipe again immediately - page changes
5. do nothing for 30 seconds 
6. swipe to change page - nothing happens
7. swipe again immediately - page changes

If "Full screen" is OFF, it works fine.
ie:
1. do nothing for 30 seconds
2. swipe to change page - page changes
3. swipe again immediately - page changes
4. swipe again immediately - page changes
5. do nothing for 30 seconds 
6. swipe to change page - page changes
7. swipe again immediately - page changes


```

Original comment by `lemons...@gmail.com` on 28 Nov 2012 at 11:18

**Comment by GoogleCodeExporter** [comment]
```
It is a normal behaviour for Android 4.x

Partially fixed only in CM10 builds.
```

Original comment by `Andrei.K...@gmail.com` on 28 Nov 2012 at 12:24
